### PR TITLE
added ssh-node redirect

### PR DESCRIPTION
**Pull request by sascala**

### CODE REVIEW

File: dcos-docs [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 0c019df900bb4434cfdad91dbb0918203dbf01be
```
Comment (joel-hamill) [inline dcos-docs]: did you mean to bump the docs?

**Comment by sascala** [comment]
No!!

**Comment by joel-hamill** [comment]
@sascala can't you just back out that change from this commit?

**Comment by sascala** [comment]
Probably...

**Pull request closed without merging**